tools/apiview/parsers/CONTRIBUTING.md

```diff
@@ -45,66 +57,32 @@ Each tree node has top tokens which should be used to capture the main tokens on
-- `Tags` : Use this for opt in or opt out boolean properties. The currently supported tags are
-  - `Deprecated` Mark a node as deprecated
-  - `Hidden`  Mark a node as Hidden
-  - `HideFromNavigation` Indicate that anode should be hidden from the page navigation.
-  - `SkipDiff` Indicate that a node should not be used in computatation of diff.
-  - `CrossLangDefId` The cross language definitionId for the node.
-- `Properties` : Use this for other properties of the node. The currently supported keys are
-  - `SubKind` Similar to kind, use this to make the node more specific. e.g.   `Kind = 'Type'`, and `SubKind = 'class'` or somethign specific to you language. We also use this to make the navigation icon it will overide kind.
-  - `IconName` Use this only if you are looking to add a custom icon different from language wide defaults. New addtions will need to be supported APIView side.
-- `TopTokens` : The main data of the node. This is all the tokens that actually define the node. e.g. For a class this would include the access modifier, the class name, any attributes or decorators e.t.c. For a method this would include the return type, method name, parameters e.t.c. See StructuredToken description below for more info.
-- `BottomToken` : Data that closes out the node. Depending on the language this would include the closing curly brace and/or empty lines. You can simulate an empty line by adding an empty token (content token with no value) and a lineBreak token.
-- `Children` : The nodes immediate children. For a namespace this would be classes, for a class this would be the class constructors and methods.
-
 
-Sort each node at each level of the tree by your desired property, this is to ensure that difference in node order does not result in diff.
+- Ensure each node has an Id and Kind. The combination of `Id`, `Kind`, and `SubKind` should make the node unique across all nodes in the tree. For example a class and a method can potentially have the same Id, but the kind should differentiate them from each other.
+- Sort each node at each level of the tree by your desired property, this is to ensure that difference in node order does not result in diff.
 
-Ensure each node has an Id and Kind. The combination of `Id`, `Kind` and `SubKind` should make the node unique across all nodes in the tree. This is very important. For example a class and a method can potentally have the same Id, but the kind should diffrentiate them from each other.
+### StructuredToken
 
-Dont worry about indentation that will be handeled by the tree structure, unless you want to have indentation between the tokens then use `TabSpace` token kind.
+- Assign the final parsed value to a `List<APITreeNode> APIForest` property of the `CodeFile`
 
+## Serialization
 
-### StructuredToken
-- `Value` : The token value which will be dispalyed. Spacing tokens don't need to have value.
```

Comment:
Unless the plan is to actually render the doc comments to markdown, removing this information is a net negative. Parser authors should not have to be looking through C# code for documentation.

Reply:
Understood. I am looking into it. But this way the documentation keeps updated with the code. We will keep improving this